packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -50,7 +50,10 @@ enum DismissDirection {
   up,
 
   /// The [Dismissible] can be dismissed by dragging down only.
-  down
+  down,
+
+  /// The [Dismissible] can be made non-dismissible, behaving like a normal widget.
```

Comment:
`The [Dismissible] cannot be dismissed by dragging.`

Reply:
Made the changes. Changed return value of ```_extentToDirection()``` from null to ```DismissDirection.none```, modified ```DismissDirection?``` to ```DismissDirection``` wherever necessary and removed unnecessary asserts.